### ISSUE TITLE
opam config env is toally broken for fish

### ISSUE THREAD

**Issue by cartazio** [ocaml/opam#3189]
my work around is 
```
set -x SHELL fish
alias opam-fix="echo (opam config env) | sed -e 's/\"/\ /g' | . "
opam-fix ;


```
in my config.fish file

i'm using version 2.6 of fish on Mac OSX

**Comment by AltGr** [comment]
What version of opam ? This has been fixed like ten times since 1.2.2

**Comment by AltGr** [comment]
Closing as this seems fixed already on master
